assets/css/style.scss

```diff
@@ -167,3 +167,12 @@ pre code {
   display: inline-block;
   box-shadow: none;
 }
+#contributors td{
+  min-width: 130px;
```

Comment:
I think that was a great solution to make it look nice on mobile. :slightly_smiling_face:

Reply:
Thanks @sglavoie 😅